### ISSUE TITLE
The value of the loss after training 28000 iters

### ISSUE THREAD

**Issue by aikudexiaohai** [wy1iu/sphereface#80]
At the beginning, the loss is about 9.x, After training 28,000 times, how much is the loss value. My loss value is approximately 4.x, the accuracy is 91%, and I try to reduce lr, but the loss also didn't ruduce

**Comment by zuoqing1988** [comment]
Try many times.  
Loss is around 1.0, best acc is around 99.2x%

**Comment by wy1iu** [comment]
Not sure about which dataset you use. If you are using different dataset other than CASIA, the loss will likely be different. You need to adjust the hyperparameters.

**Comment by Gerkam** [comment]
@wy1iu @zuoqing1988 Did you use washed normal Casia-Webface or cleaned Casia-Webface?
